integration-tests/azure-storage-blob/src/main/resources/application.properties

```diff
@@ -0,0 +1,27 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#
+
+
+# Azurite test account credentials
+azurite.storage.account.name = devstoreaccount1
+azurite.storage.account.key = Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+
+camel.component.azure-blob.credentials-account-name = ${AZURE_STORAGE_ACCOUNT_NAME:${azurite.storage.account.name}}
+camel.component.azure-blob.credentials-account-key = ${AZURE_STORAGE_ACCOUNT_KEY:${azurite.storage.account.key}}
+
+#quarkus.native.additional-build-args = --allow-incomplete-classpath,--report-unsupported-elements-at-runtime
```

Comment:
Just a nit - I guess this is redundant and can be removed (in a later commit)?

Reply:
Done in https://github.com/apache/camel-quarkus/pull/2143 , thanks for noticing!